AuroraStudiosDigipen/Borealis
# C#scripting

Implemented C# scripting basic interface for both native classes and C# monobehaviour

To test:
Build project and run, there should be an entity inside the scene hierarchy by default
Right click scene hierarchy panel and create an empty entity
Add a camera component to the entity via the inspector on the right
Click the O button on the top menu above the viewport beside the camera selection

In terminal: it should have printed that
Script is loaded
Script started
Script is running/updating(every frame)
